Changelog.md

```diff
@@ -3,6 +3,7 @@
 - Adds `response` computed property to `Error` type, which yields a Response object if available.
 - Added URLEncodedInURL to ParameterEncoding.
 - Adds convenience `endpointByAdding` method.
+- **Breaking Change** Remove our own implementation of `ParameterEncoding` and make it a `public typealias` of `Alamofire.ParameterEncoding`.
```

Comment:
Is it a breaking change? No one could have been relying on the underlying value of our enum because it didn't _have_ one, and we've kept the name the same, so existing extensions anyone had would be fine. Thoughts?